go.mod

```diff
@@ -33,8 +33,8 @@ require (
 	google.golang.org/api v0.226.0
 	google.golang.org/protobuf v1.36.6
 	gopkg.in/go-playground/validator.v9 v9.31.0
-	istio.io/api v1.24.2
-	istio.io/client-go v1.24.2
+	istio.io/api v1.27.1
```

Comment:
shouldn't we do this on a different PR?

Reply:
correct. This pr focus on only installatio scripts part.